external/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ string(REPLACE "-mfpu=fpv4-sp-d16" "" MODIFIED_C_FLAGS ${MODIFIED_C_FLAGS_TMP})
 # wally-core
 
 # configure flags for secp256k1 bundled in libwally core, to reduce memory consumption
-set(LIBWALLY_SECP256k1_FLAGS --with-ecmult-window=2 --with-ecmult-gen-precision=2 --enable-ecmult-static-precomputation)
+set(LIBWALLY_SECP256k1_FLAGS --with-ecmult-window=2 --with-ecmult-gen-precision=2 --enable-ecmult-static-precomputation --enable-module-ecdsa-s2c)
```

Comment:
I'm probably missing something because the CI builds this ok, but it seems like `enable-module-ecdsa-s2c` flag was introduced in https://github.com/ElementsProject/secp256k1-zkp/commit/8e46cac. We're using https://github.com/digitalbitbox/libwally-core/tree/bitbox02-firmware-20201213 in which secp256 lib is at an earlier commit [6370bdd](https://github.com/ElementsProject/secp256k1-zkp/tree/6370bdd537492638e5ae0f7235cc73daab13ff12).

How does it work?

---

aha wait, `git log external/libwally-core` was updated by 305f78c5b75eb2e3a048aceaba31da4dc7424158:

![image](https://user-images.githubusercontent.com/25405/103769509-b58ea000-5024-11eb-9026-03ec5bbeae09.png)

 So, we're now actually using a libwally at a new branch `bitbox02-20210104` where libsecp is switched to https://github.com/digitalbitbox/secp256k1-zkp. Just a bit confusing, can't remember this happening.

Is the plan to send https://github.com/digitalbitbox/secp256k1-zkp/commit/5e3bd34a5eb0409f5c5f42f3c3385d7ebfcc15dc upstream and will this be our little custom changes?

Reply:
About that .... see my previous comment:

https://github.com/digitalbitbox/bitbox02-firmware/pull/697#issuecomment-754973416

Sorry for that :innocent:

Reply:
ah those emacs users... :laughing: